src/js/classes/AjaxBootstrapSelectList.js

```diff
@@ -284,6 +284,12 @@ AjaxBootstrapSelectList.prototype.replaceOptions = function (data) {
 
     // Replace the options.
     this.plugin.$element.html(output);
+
+    // Collapse dropdown after large search results
+    this.plugin.$element.nextAll('.dropdown-menu')
+        .find('ul.dropdown-menu.inner')
+        .css({ margin: 0 });
```

Comment:
I'm not a huge fan of using jQuery/JavaScript to style elements. This almost feels like fixing a symptom rather than the actual cause.